src/EnergyPlus/DataSurfaces.hh

```diff
@@ -1780,6 +1792,7 @@ struct SurfacesData : BaseGlobalStruct
     EPVector<DataSurfaces::SurfaceLocalEnvironment> SurfLocalEnvironment;
     EPVector<DataSurfaces::SurroundingSurfacesProperty> SurroundingSurfsProperty;
     EPVector<DataSurfaces::IntMassObject> IntMassObjects;
+    EPVector<DataSurfaces::GroundSurfacesProperty> GroundSurfsProperty;
```

Comment:
EPVector is fine here, although whenever possible, I would suggest pushing toward the raw vector for new code.

Reply:
@Myoldmopar I will address your comments but I don't have additional changes on my side.